pyproject.toml

```diff
@@ -42,16 +42,12 @@ faker = ">=4.1,<12.0"
 pytest-xdist = "*"
 filelock = "^3.0"
 pytest-ordering = "*"
-lovely-pytest-docker = { version="^0", optional = true }
 junitparser = "^2.2.0"
 addonfactory-splunk-conf-parser-lib = "^0.3.3"
+crudini = "0.9.3"
```

Comment:
You do not need `cruduni` here, please checkout `addonfactory-splunk-conf-parser-lib`, it does pretty much the same thing you need.